tensorflow/compiler/xla/service/gpu/miopen_conv_algorithm_picker.cc

```diff
@@ -0,0 +1,339 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
@sumesh13 looking at the source code of this new optimization pass... could you elaborate the difference between its `cudnn` counterpart? I could imagine we'll be challenged with the same question when we upstream this pass.

Reply:
Will add comment in the file